src/ui/zcl_abapgit_gui.clas.abap

```diff
@@ -345,4 +373,26 @@ CLASS ZCL_ABAPGIT_GUI IMPLEMENTATION.
     SET HANDLER me->on_event FOR mo_html_viewer.
 
   ENDMETHOD.                    "startup
+
+  METHOD remove_dummy_action.
```

Comment:
This the right place for documenting what the method does and why it even exists.

Reply:
yep good point, will do

Reply:
Ok, I refactored this heavily.
this method doesn't exist anymore, and the new transform_dummy_form_params is documented much better